tests/installed/destructive.yml

```diff
@@ -0,0 +1,35 @@
+# This entrypoint right now just runs the sysinstalled-tests.
+---
+- hosts: localhost
+  tags:
+  - atomic
+  remote_user: root
+  vars:
+    use_git_build: True
+    tests: "."
```

Comment:
So this can be used to select which tests to run, right? Can you document that in the `README.md`?